### ISSUE TITLE
`tcp 16-20` add more endpoints for DigitalOcean

### ISSUE THREAD

**Issue by hyperion-cs** [hyperion-cs/dpi-checkers#39]
Currently, only one endpoint is used, and it does not provide accurate enough results for the entire provider.

**Comment by its0ka** [comment]
я не понимаю почему
https://hyperion-cs.github.io/dpi-checkers/ru/tcp-16-20/?url=https://cdn.fastvalue.vn
пишет Stream ended but data is too small и possibly detected на сети без блока

я нашел много DO сайтов и проверил десяток в тестере, но not detected ни разу не написано
https://pastebin.com/raw/8tScjPrD

**Comment by hyperion-cs** [comment]
@its0ka this is actually a bug, and it has been fixed [here](https://github.com/hyperion-cs/dpi-checkers/commit/c3571004020a8ca2229ed51f0024a7095b5118d4). Please note that a new `thrBytes` parameter has been added for custom endpoints, and it should be evaluated as described in the [readme](https://github.com/hyperion-cs/dpi-checkers#checkers-list) (see _tcp 16-20_ section). For example, after checking the results of utility "http compression prober" for your endpoint:
```sh
$ python http_compression_prober.py --url https://cdn.fastvalue.vn/
gzip: compr=24576 decompr=204342 (x8.31)
deflate: not supported by endpoint
br: compr=24576 decompr=222726 (x9.06) <-- max
zstd: compr=24576 decompr=193639 (x7.88)
```

It can be understood that a `thrBytes` value of at least `222726` should be used. So:
https://hyperion-cs.github.io/dpi-checkers/ru/tcp-16-20/?url=https://cdn.fastvalue.vn&thrBytes=222726
should work for you.

---
P.S. ⚠️ You should submit such questions and information as **separate** issues.

**Comment by hyperion-cs** [comment]
Closed in https://github.com/hyperion-cs/dpi-checkers/commit/c9eeb76381c781af7da19b6842cc3023a3d14fc9.
Thanks to @its0ka for preparing a list of DigitalOcean resources; new endpoints were selected based on this list.
